diffpy/snmf/correlation.py

```diff
@@ -1,19 +1,23 @@
 import numpy as np
 
 
-def pearson(data):
+def pearson(data1, data2):
     """
+    Returns the pearson correlation coefficient between two data PDF/XRD patterns
 
     Parameters
     ----------
-    data
-
+    data1: ndarray
```

Comment:
I think you can take a list or an array, right?

Reply:
Yes. I will update the docstring